.github/workflows/test.yml

```diff
@@ -0,0 +1,75 @@
+name: lint-test
+
+on:
+  pull_request:
+  schedule:
+    # run once a week on early monday mornings
+    - cron: '22 2 * * 1'
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r dev-requirements.txt
+      - name: Test with tox
+        run: tox
```

Comment:
Invoking tox without any arguments will run every combination of python version and django version as set in tox.ini, so I'm guessing combining that with github actions' own python version matrix could result in some redundant test runs.

Reply:
Ahh yes, this is what the [tox-gh-actions](https://pypi.org/project/tox-gh-actions/) package does (installed in dev-requirements.txt). It specifically works with gh-actions to only run the toxenvs that match the current github-action python version. We configure it by a `[gh-actions]` section in tox.ini which maps github-action python versions to tox python versions. (That was confusing to type, so happy to talk it over). Also open to adding more docs about that, in the README perhaps?

Reply:
I didn't notice the "[gh-actions]" section in tox.ini at all :-(. Maybe a comment in the github workflow configuration near the matrix?